### DISCUSSION TITLE
Can we zip the ToC file?

### DISCUSSION THREAD

**Discussion by Nio-1** [CMSgov/price-transparency-guide#532] [Q&A] [answered]
We expect our ToC file to grow in size.  Are we allowed to zip the Toc file and publish the zipped URL for the ToC?  
Example: 2020-01-05_cms_index.json.gz  @shaselton-usds

**Comment by mkress-epic** [comment, marked as answer]
https://github.com/CMSgov/price-transparency-guide/discussions/59

"There are no restrictions in zipping the files."
